<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <widget class="QRadioButton" name="radioCircular">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>106</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>RadioButton</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioRectangular">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>106</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>RadioButton</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelPulv">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>100</y>
          <width>64</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelDiam">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>180</y>
          <width>64</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelAncho">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>250</y>
          <width>64</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelLargo">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>290</y>
          <width>64</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonCalc">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>350</y>
          <width>83</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditPulv">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>100</y>
          <width>113</width>
          <height>23</height>
         </rect>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhDigitsOnly</set>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditDiam">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>180</y>
          <width>113</width>
          <height>23</height>
         </rect>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhDigitsOnly</set>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditAncho">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>250</y>
          <width>113</width>
          <height>23</height>
         </rect>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhDigitsOnly</set>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditLargo">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>290</y>
          <width>113</width>
          <height>23</height>
         </rect>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhDigitsOnly</set>
        </property>
       </widget>
       <widget class="QTextEdit" name="textEditPulv">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>110</y>
          <width>104</width>
          <height>71</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QGraphicsView" name="graphicsView"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
